src/pool-cl/libraries/CLPool.sol

```diff
@@ -145,7 +142,7 @@ library CLPool {
         }
 
         // Fees earned from LPing are removed from the pool balance.
-        feeDelta = toBalanceDelta(feesOwed0.toInt128(), feesOwed1.toInt128());
+        feeDelta = toBalanceDelta(-feesOwed0.toInt128(), -feesOwed1.toInt128());
```

Comment:
feeDelta must be negative i.e. both `feeDelta.amount0 <= 0` and `feeDelta.amount1 <= 0`

Because that's the fee owed to the user by the pool. 

If we decide to flip to sign, we'd better do it for all the delta value